// public class TryCatchExample3 {
//     public static void main(String[] args) {
        
//         int i = 50;
//         int j = 0;
//         int data;
//         try{
//           data = i/j;
//         }
//         catch(ArrayIndexOutOfBoundsException e){
//             System.out.println(e);
//         }


//     }
    
// }
